### PR TITLE
common Logger sanitizer 

### PR DESCRIPTION
**Pull request by sar-perlman**

### CODE REVIEW

File: logger-sanitizer/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const _isArray = require('lodash/isArray');
+const _isPlainObject = require('lodash/isPlainObject');
+
+// Replaces password info in a logging message with ['redacted']
+const loggerSanitizer = (data) => {
+
+  // define list of keys to sanitize
+  const blacklistedLogKeys = ['pass', 'password', 'pw', 'token', 'credit', 'card', 'cc'];
```
Comment (Jacfem) [inline logger-sanitizer/index.js]: Were these taken from real examples of keys we've previously passed?

File: logger-sanitizer/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    const testLog = {
+      user: 'tester',
+      password: '12345',
+      data: {
+        pass: '12345',
+        cc: '2345-4242-4242-4242'
+      },
+      card: '2345-4242-4242-4242',
+      token: '345678',
+      happyKey: 'I am a happy key'
```
Comment (Jacfem) [inline logger-sanitizer/index.spec.js]: 😁

File: logger-sanitizer/index.spec.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    const data = loggerSanitizer(testLog);
+    expect(data.user).toEqual('tester');
+    expect(data.happyKey).toEqual('I am a happy key');
+  });
+  it('Should not redact a string', () => {
+    const testLog = 'happyString';
+    const data = loggerSanitizer(testLog);
+    expect(data).toEqual('happyString');
+  });
+  it('Should not filter arrays unless the array value is an object', () => {
```
Comment (Jacfem) [inline logger-sanitizer/index.spec.js]: Nice!

File: logger-sanitizer/index.js (hunk 1 above)
Comment (sar-perlman) [reply logger-sanitizer/index.js]: Only `password` was a real example. Rest are potential keys.

**Pull request merged**